sdk/python/requirements/py3.10-requirements.txt

```diff
@@ -89,9 +89,7 @@ partd==1.4.2
 prometheus-client==0.20.0
     # via feast (setup.py)
 protobuf==4.25.4
-    # via
-    #   feast (setup.py)
-    #   mypy-protobuf
+    # via mypy-protobuf
```

Comment:
I still think we should also investigate whether `mypy-protobuf` should also be relegated as a dev dependency or not.. but I'm fine with postponing that as well

Reply:
We can postpone that.